### PR TITLE
[Feat] Add Workspace Administrator and Workspace Manager roles documentation

### PR DESCRIPTION
**Pull request by dragon-slayer875**
**Notes for Reviewers**

This PR adds documentation for the workspace administrator and workspace manager roles.

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by dragon-slayer875** [comment]
- [ ] Need to add page that shows the list of all available keys
- [ ] Need to update images

**Review by leecalcote — CHANGES_REQUESTED** [review]
The graphic in the SVG still denotes these new roles as "Future Roles"

**Comment by Yashsharma1911** [comment]
Thankyou for adding it

**Review by Yashsharma1911 — APPROVED** [review]
LGTM, after image update we can merge this

File: content/en/cloud/security/roles/_index.md [hunk 1]
```diff
@@ -153,15 +218,15 @@ Min: 1, Max: many
 
 - Organization Administrators or Team Owner
 
-**When this role first assigned?**
+**When this role is first assigned?**
```
Comment (leecalcote) [inline content/en/cloud/security/roles/_index.md:221]:
```suggestion
**When is this role first assigned?**
```

File: content/en/cloud/security/roles/_index.md [hunk 2]
```diff
@@ -135,7 +200,7 @@ For more information, see [Organization](/cloud/identity/organizations).
 
 - Organization Administrator or Team owner
 
-**When this role first assigned?**
+**When this role is first assigned?**
```
Comment (leecalcote) [inline content/en/cloud/security/roles/_index.md:203]:
```suggestion
**When is this role first assigned?**
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**